gui/idact_app.py

```diff
@@ -8,6 +8,7 @@
 Ui_AddNativeArgument, QtBaseClass = uic.loadUiType('../widgets_templates/add-native.ui')
 Ui_RemoveNativeArgument, QtBaseClass = uic.loadUiType('../widgets_templates/remove-native.ui')
 Ui_ShowNativeArgument, QtBaseClass = uic.loadUiType('../widgets_templates/show-native.ui')
+Ui_Error, QtBaseClass = uic.loadUiType('../widgets_templates/error.ui')
```

Comment:
Could you change all `QtBaseClass` into `_QtBaseClass`? I think it is the convention of naming not used variables.

Reply:
Done.